fuzz/fuzz.py

```diff
@@ -0,0 +1,31 @@
+#!/bin/python3
```

Comment:
Only request here is to change the name of the file to something that describes what is being fuzzed. You would want to apply the change in the Mayhemfile target name as well.